index.html

```diff
@@ -1876,108 +1806,101 @@ <h2>Verification Methods</h2>
-        </p>
+        <h3>Verification Material</h3>
 
         <p>
-If the value of a <a>verification method</a> property is a
-<a data-cite="INFRA#ordered-map">map</a>, the <a>verification method</a> has
-been embedded and its properties can be accessed directly. However, if the
-value is a URL <a data-cite="INFRA#string">string</a>, the <a>verification
-method</a> has been included by reference and its properties will need to be
-retrieved from elsewhere in the <a>DID document</a> or from another <a>DID
-document</a>. This is done by dereferencing the URL and searching the
-resulting resource for a <a>verification method</a>
-<a data-cite="INFRA#ordered-map">map</a> with an <code>id</code> property
-whose value matches the URL.
-        </p>
-
-        <pre class="example nohighlight"
-        title="Embedding and referencing verification methods">
-{
-<span class="comment">...</span>
-
-  "authentication": [
-    <span class="comment">// this key is referenced, it may be used with more than one verification relationship</span>
-    "did:example:123456789abcdefghi#keys-1",
-    <span class="comment">// this key is embedded and may *only* be used for authentication</span>
-    {
-      "id": "did:example:123456789abcdefghi#keys-2",
-      "type": "Ed25519VerificationKey2018", <span class="comment">// external (property value)</span>
-      "controller": "did:example:123456789abcdefghi",
-      "publicKeyBase58": "H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
-    }
-  ],
-
-<span class="comment">...</span>
-}
-        </pre>
-      </section>
-
-      <section>
-        <h3>Verification Method types</h3>
-
-        <p>
-A public key can be used as a <a>verification method</a>.
+Verification material is defined as any information that is used by a process
+that applies a <a>verification method</a> and is typically determined by the
+value of the <a>verification method</a>'s <code>type</code> property. Examples
+of verification material properties are <code><a>publicKeyJwk</a></code> or
+<code><a>publicKeyBase58</a></code>. A verification suite definition is
+responsible for specifying the <a>verification method</a> <code>type</code> and
+its associated verification material. For example, see <a
+href="https://w3c-ccg.github.io/lds-jws2020/">JSON Web Signature 2020</a> and <a
+href="https://w3c-ccg.github.io/lds-ed25519-2018/">Ed25519 Signature 2018</a>.
+For all registered <a>verification method</a> types and associated  verification
+material available for <a>DIDs</a>, please see the DID Specification Registries
+[[?DID-SPEC-REGISTRIES]].
         </p>
 
         <p>
-This specification strives to limit the number of formats for expressing public
-key material in a <a>DID document</a> to the fewest possible, to increase the
-likelihood of interoperability. The fewer formats that implementers have to
+In order to increase the likelihood of interoperability, this specification
```

Comment:
```suggestion
To increase the likelihood of interoperable implementations, this specification
```
`interoperable implementations` _could_ be left as just `interoperability`, but I think this makes clearer what it is we're wanting to be interoperable.